Samples/SPM/SPM/SPMTestApp.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SPMTestApp.swift
+//  SPMTest
+//
+//  Created by Noah Martin on 10/15/25.
+//
+
+import SentrySwift
+import SwiftUI
+
+@main
+struct SPMTestApp: App {
+  init() {
+    let options = Options()
+    options.enableAppHangTracking = true
+    options.dsn = "testing"
```

Comment:
`m`: I believe we should set up all samples with the sentry-sdks DSN, you can use this for reference:

https://github.com/getsentry/sentry-cocoa/blob/4606e3591f4ad082b2dfb2dc876cdf4f6ea9bd2b/Samples/iOS-SwiftUI-Widgets/iOS-SwiftUI-Widgets/Sources/MainApp.swift#L10-L19